scrape/scrape.go

```diff
@@ -1273,8 +1280,30 @@ loop:
 		if ok {
 			err = app.AddFast(ce.ref, t, v)
 			_, err = sl.checkAddError(ce, met, tp, err, &sampleLimitErr, &appErrs)
+			switch err {
+			case nil:
+				if tp == nil {
+					sl.cache.trackStaleness(ce.hash, ce.lset)
+				}
+				// todo: This smells funny.
+				if hasExemplar := p.Exemplar(&e); hasExemplar {
+					if err := exemplarApp.AddExemplar(ce.lset, t, e); err != nil {
+						if err == storage.ErrDuplicateExemplar {
+							level.Debug(sl.l).Log("msg", "Duplicate exemplar", "seriesLabels", ce.lset, "exemplar", e)
```

Comment:
This is not an error, so shouldn't be logged - or probably even reported as an error by AddExemplar.

Reply:
How is it not an error?

Reply:
If there's been no events since the last scrape, all the exemplars will be the same. That's not an error.

Reply:
Yes, I think it should not be reported at all in the first place.

Reply:
We could have an error if the same exemplar id is exposed with 2 different timestamps.